tests/testthat/_snaps/survival-tune_race_anova.md

```diff
@@ -1,6 +1,6 @@
 # race tuning (anova) survival models with integrated metric
 
-    Evaluation times are only required when dynmanic or integrated metrics are selected as the primary metric (and will be ignored).
+    An evaluation time is only required when a dynamic metric is selected (and `eval_time` will thus be ignored).
```

Comment:
Can we add the call to this warning?

Reply:
changes in tidymodels/tune#806; tests updates here in a few min

Reply:
updated snapshots in [43f2cd1](https://github.com/tidymodels/extratests/pull/158/commits/43f2cd11d5e5e00e2f415eec81706f52aa260294)